docs: add example of using Circle Paymaster middleware (#1347)
alchemyplatform/aa-sdk: site/pages/third-party/paymasters.mdx

We are working on building support for an ERC-20 paymaster!

> To allow users to pay for gas permissionlessly using USDC, you can use [Circle Paymaster](https://developers.circle.com/stablecoins/paymaster-overview). Here's an example using Circle Paymaster with the Alchemy SDK:

Reviewer: Let's update this to:

```suggestion
If you are looking to pay for gas permissionlessly using USDC, you can use [Circle's Paymaster](https://developers.circle.com/stablecoins/paymaster-overview). Here's an example using Circle Paymaster with the Alchemy SDK:
```
